tests/generators/bls/main.py

```diff
@@ -69,8 +69,8 @@ def case02_verify():
             # Valid signature
             signature = bls.G2ProofOfPossession.Sign(privkey, message)
             pubkey = bls.G2ProofOfPossession.PrivToPub(privkey)
-            full_name = f'{encode_hex(pubkey)}_{encode_hex(message)}_valid'
-            yield f'verify_case_{(hash(bytes(full_name, "utf-8"))[:8]).hex()}', {
+            full_name = f'{encode_hex(pubkey)}_{encode_hex(message)}'
```

Comment:
nitpick: What do you think renaming all the `full_name`s to `identifier`? They are no longer the full name actually.

Reply:
will do!